core/shared/src/main/scala/fs2/Interrupted.scala

```diff
@@ -0,0 +1,18 @@
+package fs2
+
+import fs2.internal.Token
+
+/**
+  * Signals interruption of the evaluation. Contains id of last scope that shall be interrupted and
+  * any children of that scope.
```

Comment:
perhaps we can reword a bit?
"Contains id of last scope that shall be interrupted along with its children"

Assuming that the meaning is "this will interrupt the scope with id scopeId and also interrupt all the children of said scope"

Reply:
ok